kos_operator/crds.py

```diff
@@ -321,6 +321,7 @@ def execute(self, state, variables):
             LOG.debug('exec code for {}'.format(self.name))
             exec(self.code, variables, local_vars)
         except Exception as e:
+            LOG.exception("Error executing query")
```

Comment:
Should we also add `self.name` to the message so we know which code was executed? We might not always have debug-logging enabled so might not see the `LOG.debug()` above.

Reply:
Which code is executed is visible in the backtrace, thanks to the "filename" being passed to `compile`.

It looks like this
```
Traceback (most recent call last):
  File "/sapmnt/home3/d063079/repos/kos-operator/kos_operator/crds.py", line 322, in execute
  exec(self.code, variables, local_vars)
  File "ironic-node-update", line 137, in <module>
  File "ironic-node-update", line 83, in filtered_patch
NameError: name '_filtered_patch' is not defined
```

Well, technically, the name also contains the namespace, but I'd rather then change the file to also include that as path.

Reply:
Thank you for providing these detailed information.

Reply:
I've added the namespace to the filename.